go.mod

```diff
@@ -25,7 +25,7 @@ replace (
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc92
 	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.3-0.20200728170252-4d89ac9fbff6
-	go.etcd.io/etcd => github.com/rancher/etcd v0.5.0-alpha.5.0.20200911210206-f8fde3601008
+	go.etcd.io/etcd => github.com/galal-hussein/etcd v0.5.0-alpha.5.0.20201124000623-1b92ee32962b
```

Comment:
Are you planning to push these to rancher/etcd or k3s-io/etcd?

Reply:
I am not sure what is the plan regarding the changes to k3s-io repos, should we use k3s-io instead now?